apps/meeting/src/containers/EndMeetingControl/index.tsx

```diff
@@ -54,18 +54,20 @@ const EndMeetingControl: React.FC = () => {
           <ModalButtonGroup
             primaryButtons={[
               <ModalButton
+                key="end-meeting-for-all"
                 onClick={endMeetingForAll}
                 variant="primary"
                 label="End meeting for all"
                 closesModal
               />,
               <ModalButton
+                key="leave-meeting"
                 onClick={leaveMeeting}
                 variant="primary"
                 label="Leave Meeting"
                 closesModal
               />,
-              <ModalButton variant="secondary" label="Cancel" closesModal />
+              <ModalButton key="cancel-meeting-ending" variant="secondary" label="Cancel" closesModal />
```

Comment:
Need to change double quote to single quote

Reply:
HTML or JSX props are still preferred to be double quotes over single.

Reply:
For giggle https://github.com/airbnb/javascript/issues/629 :D

Reply:
Lol, there should be JSX rules to auto-fix this I think. But since we are using double quotes for the props from start lets keep it that way.